src/main/java/com/javaproject/restservice/entity/User.java

```diff
@@ -0,0 +1,45 @@
+package com.javaproject.restservice.entity;
+
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+
+@Entity
+public class User {
+
+    @Id
```

Comment:
What does @Id indicate?

Reply:
The @Id annotation marks a field as a primary key field. Im indicating these are my primary keys.

Reply:
👍